wagtailio/project_styleguide/templates/patterns/pages/showcase_page/showcase_page_fragment.html

```diff
@@ -0,0 +1,57 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+<div id="showcase_results" class="showcase-grid">
+    {% for item in showcase_items %}
+        <div hx-on:click="window.location.replace('{{ item.url }}')" class="showcase-card">
+            {% if pattern_library %}
+                {# I've tried everything to get this to render using the pattern-library, #}
+                {# This is a placeholder until a fix is found. #}
+                <img src="https://placehold.co/600x400" width="600" height="400" alt="Some alt" class="showcase-card__image">
+                <img src="https://placehold.co/120x100" width="120" height="100" alt="Some alt" class="showcase-card__logo">
+            {% else %}
+                <picture>
+                    {% image item.image max-600x400 format-webp as webp_image %}
```

Comment:
If you or someone else reviews #443, we could make use of the new [`picture` template tag](https://docs.wagtail.org/en/stable/releases/5.2.html#responsive-multi-format-images-with-the-picture-tag) in 5.2